onboard/Dockerfile

```diff
@@ -21,8 +24,19 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
     # Clear apt caches to reduce image size
     rm -rf /var/lib/apt/lists/*
 
+# Downgrade to pip version 8.1.1
+RUN python -m pip install pip==8.1.1
```

Comment:
pip installed via apt should not be downgraded like this, it could lead to painful conflicts. you can either remove the pip installation from `core` and replace it with installing via `get-pip.py`, or you can find a workaround that doesn't require changing pip.

see https://github.com/pypa/pip/issues/5599